components/ShadowLayer/src/MDCShadowLayer.h

```diff
@@ -77,3 +77,10 @@
 @property(nonatomic, getter=isShadowMaskEnabled, assign) BOOL shadowMaskEnabled;
 
 @end
+
+/**
+ Subclasses can depend on MDCShadowLayer implementing CALayerDelegate actionForLayer:forKey: in
```

Comment:
Does MDCShadowLayer need to explicitly implement `actionForLayer:forKey:` or does CALayer take care of that?

Reply:
As far as I know there is no way to rely on the built-in CAAction creation for headless layers so we explicitly implement `actionForLayer:forKey:`.

Reply:
Ack. I missed the method definition in my earlier search. :-S